src/main/java/com/synopsys/integration/detect/ApplicationUpdater.java

```diff
@@ -247,25 +258,31 @@ private boolean checkInstallationDir(Path path) throws NoSuchFileException, Acce
     }
     
     private void checkEnvironmentProperties() {
-        this.blackduckHost = System.getenv(SYS_ENV_PROP_BLACKDUCK_URL);
-        final String httpsHost = System.getenv(SYS_ENV_PROP_PROXY_HTTPS_HOST);
-        final String httpHost = System.getenv(SYS_ENV_PROP_PROXY_HTTP_HOST);
+        this.blackduckHost = utility.getSysEnvProperty(SYS_ENV_PROP_BLACKDUCK_URL);
+        final String httpsHost = utility.getSysEnvProperty(SYS_ENV_PROP_PROXY_HTTPS_HOST);
+        final String httpHost = utility.getSysEnvProperty(SYS_ENV_PROP_PROXY_HTTP_HOST);
```

Comment:
The getSysEnvProperty wrapper method was created to allow mock interception for the tests.